src/status_im/ui/screens/chat/views.cljs

```diff
@@ -136,25 +135,23 @@
        :no-messages?      no-messages?}]
      [react/text {:style (assoc style/intro-header-description :margin-bottom 32)}
       (str (i18n/label :t/empty-chat-description-one-to-one) contact-name)])
-   (when (and mutual-contact-requests-enabled?
-              (= chat-type constants/one-to-one-chat-type)
-              (or (= contact-request-state constants/contact-request-state-none)
-                  (= contact-request-state constants/contact-request-state-received)
-                  (= contact-request-state constants/contact-request-state-dismissed)))
-     [contact-request])])
+   (when
+     (= chat-type constants/one-to-one-chat-type)
+     (or (= contact-request-state constants/contact-request-state-none)
+         (= contact-request-state constants/contact-request-state-received)
+         (= contact-request-state constants/contact-request-state-dismissed)))
+   [contact-request]])
 
 (defn chat-intro-one-to-one
   [{:keys [chat-id] :as opts}]
-  (let [contact                          @(re-frame/subscribe [:contacts/contact-by-identity chat-id])
-        mutual-contact-requests-enabled? @(re-frame/subscribe [:mutual-contact-requests/enabled?])
-        contact-names                    @(re-frame/subscribe [:contacts/contact-two-names-by-identity
-                                                               chat-id])]
+  (let [contact       @(re-frame/subscribe [:contacts/contact-by-identity chat-id])
+        contact-names @(re-frame/subscribe [:contacts/contact-two-names-by-identity
```

Comment:
Given you are making changes on this file from the old ns, maybe you can refactor `re-frame` require alias to `rf`, but I don't know if that would change lots of lines and make the PR more difficult to review or see the changes. Personally I'd prefer to do it now given that there is some refactoring, a little bit more couldn't hurt.